example/lib/main.dart

```diff
@@ -93,18 +100,16 @@ class _MyAppState extends State<MyApp> {
                         try {
                           response =
                               await FlutterPericulum.affordabilityAnalysis(
-                                      statementKey: 3,
-                                      dti: 2.0,
-                                      loanTenure: 30,
-                                      averageMonthlyTotalExpenses: 4000,
-                                      averageMonthlyLoanRepaymentAmount: 1000,
-                                      token: "${dotenv.env['tokenKey']}")
-                                  .then((value) => {
-                                        affordabilityResponse = value,
-                                        debugPrint(value
-                                            .averageMonthlyLoanRepaymentAmount
-                                            .toString())
-                                      });
+                                  statementKey: 3,
+                                  dti: 2.0,
+                                  loanTenure: 30,
+                                  averageMonthlyTotalExpenses: 4000,
+                                  averageMonthlyLoanRepaymentAmount: 1000,
+                                  token: "${dotenv.env['tokenKey']}");
```

Comment:
same here, can you just make this an empty string. In the readme can you please link to the authentication flow diagram in the readme?